### PR TITLE
[Doctest]fix No.163,164,166, test=docs_preview

### PR DESCRIPTION
**Pull request by liyongchao911**
### PR types
Others

### PR changes
Others

### Description
修改如下文件的示例代码，使其通过 `xdoctest` 检查：
- python/paddle/vision/models/squeezenet.py
- python/paddle/vision/models/vgg.py
- python/paddle/vision/transforms/functional.py

### Related links

-  #55629
-  #55295

@sunzhongkai588 @SigureMo @megemini

### CODE REVIEW

File: python/paddle/vision/transforms/functional.py [hunk 1]
```diff
@@ -327,16 +332,17 @@ def hflip(img):
...
-            fake_img = (np.random.rand(256, 300, 3) * 255.).astype('uint8')
+            >>> fake_img = (np.random.rand(256, 300, 3) * 255.).astype('uint8')
 
-            fake_img = Image.fromarray(fake_img)
+            >>> fake_img = Image.fromarray(fake_img)
 
-            flpped_img = F.hflip(fake_img)
-            print(flpped_img.size)
+            >>> flpped_img = F.hflip(fake_img)
+            >>> print(flpped_img.size)
```
Comment (megemini) [inline python/paddle/vision/transforms/functional.py]: flpped_img -> flipped_img

File: python/paddle/vision/transforms/functional.py [hunk 2]
```diff
@@ -368,16 +374,17 @@ def vflip(img):
...
-            fake_img = (np.random.rand(256, 300, 3) * 255.).astype('uint8')
+            >>> fake_img = (np.random.rand(256, 300, 3) * 255.).astype('uint8')
 
-            fake_img = Image.fromarray(fake_img)
+            >>> fake_img = Image.fromarray(fake_img)
 
-            flpped_img = F.vflip(fake_img)
-            print(flpped_img.size)
+            >>> flpped_img = F.vflip(fake_img)
+            >>> print(flpped_img.size)
```
Comment (megemini) [inline python/paddle/vision/transforms/functional.py]: flpped_img -> flipped_img

File: python/paddle/vision/transforms/functional.py [hunk 3]
```diff
@@ -411,20 +418,27 @@ def adjust_brightness(img, brightness_factor):
...
-            fake_img = (np.random.rand(256, 300, 3) * 255.).astype('uint8')
+            >>> fake_img = (np.random.rand(256, 300, 3) * 255.).astype('uint8')
+            >>> fake_img = Image.fromarray(fake_img)
+
+            >>> print(fake_img.load()[1,1])
+            (95, 127, 202)
+            >>> converted_img = F.adjust_brightness(fake_img, 0.5)
+            >>> print(converted_img.load()[1,1])
+            (47, 63, 101)
+            >>> # doctest: -SKIP
```
Comment (megemini) [inline python/paddle/vision/transforms/functional.py]: 用 seed，不需要 skip ～

File: python/paddle/vision/transforms/functional.py [hunk 4]
```diff
@@ -946,16 +966,16 @@ def to_grayscale(img, num_output_channels=1):
...
-            fake_img = (np.random.rand(256, 300, 3) * 255.).astype('uint8')
+            >>> fake_img = (np.random.rand(256, 300, 3) * 255.).astype('uint8')
 
-            fake_img = Image.fromarray(fake_img)
+            >>> fake_img = Image.fromarray(fake_img)
 
-            gray_img = F.to_grayscale(fake_img)
-            print(gray_img.size)
+            >>> gray_img = F.to_grayscale(fake_img)
+            >>> print(gray_img.size)
```
Comment (megemini) [inline python/paddle/vision/transforms/functional.py:936]: 输出～

File: python/paddle/vision/transforms/functional.py [hunk 5]
```diff
@@ -993,19 +1013,20 @@ def normalize(img, mean, std, data_format='CHW', to_rgb=False):
...
-            mean = [127.5, 127.5, 127.5]
-            std = [127.5, 127.5, 127.5]
+            >>> mean = [127.5, 127.5, 127.5]
+            >>> std = [127.5, 127.5, 127.5]
 
-            normalized_img = F.normalize(fake_img, mean, std, data_format='HWC')
-            print(normalized_img.max(), normalized_img.min())
+            >>> normalized_img = F.normalize(fake_img, mean, std, data_format='HWC')
+            >>> print(normalized_img.max(), normalized_img.min())
+            (300, 256)
```
Comment (megemini) [inline python/paddle/vision/transforms/functional.py]: 输出不对～

File: python/paddle/vision/transforms/functional.py [hunk 6]
```diff
@@ -1039,35 +1060,33 @@ def erase(img, i, j, h, w, v, inplace=False):
...
+
+            Tensor(shape=[3, 2, 4], dtype=float32, place=Place(gpu:0), stop_gradient=True,
+            [[[ 0.02169025,  0.        ,  0.        , -1.07478464],
+            [ 0.20654772,  1.74624777,  0.32268861, -0.13857445]],
+
+            [[-0.14993843,  0.        ,  0.        , -1.94395220],
+            [ 0.41686651,  0.44551995, -0.09356714, -0.60898107]],
+
+            [[-0.24998808,  0.        ,  0.        ,  0.42629015],
+            [ 0.56948012, -0.96200180,  0.53355658,  3.20450878]]])
```
Comment (megemini) [inline python/paddle/vision/transforms/functional.py]: 用 seed ～ 另外，最好把方括号对齐一下 ～

File: python/paddle/vision/transforms/functional.py (hunk 1 above)
Comment (liyongchao911) [reply python/paddle/vision/transforms/functional.py]: done!

File: python/paddle/vision/transforms/functional.py (hunk 2 above)
Comment (liyongchao911) [reply python/paddle/vision/transforms/functional.py]: done!

File: python/paddle/vision/transforms/functional.py (hunk 6 above)
Comment (liyongchao911) [reply python/paddle/vision/transforms/functional.py]: done！

File: python/paddle/vision/transforms/functional.py (hunk 3 above)
Comment (liyongchao911) [reply python/paddle/vision/transforms/functional.py]: done

File: python/paddle/vision/transforms/functional.py (hunk 4 above)
Comment (liyongchao911) [reply python/paddle/vision/transforms/functional.py:936]: done

File: python/paddle/vision/transforms/functional.py (hunk 5 above)
Comment (liyongchao911) [reply python/paddle/vision/transforms/functional.py]: done

File: python/paddle/vision/transforms/functional.py [hunk 7]
```diff
@@ -993,19 +1012,20 @@ def normalize(img, mean, std, data_format='CHW', to_rgb=False):
...
-            mean = [127.5, 127.5, 127.5]
-            std = [127.5, 127.5, 127.5]
+            >>> mean = [127.5, 127.5, 127.5]
+            >>> std = [127.5, 127.5, 127.5]
 
-            normalized_img = F.normalize(fake_img, mean, std, data_format='HWC')
-            print(normalized_img.max(), normalized_img.min())
+            >>> normalized_img = F.normalize(fake_img, mean, std, data_format='HWC')
+            >>> print(normalized_img.max(), normalized_img.min())
+            (0.99215686 -1.0)
```
Comment (SigureMo) [inline python/paddle/vision/transforms/functional.py]: image

**Review by SigureMo — APPROVED** [review]
自行看下 log 吧，还有很多对不上的

**Review by SigureMo — CHANGES_REQUESTED** [review]
点错了，不应该 approve 的

File: python/paddle/vision/transforms/functional.py [hunk 8]
```diff
@@ -411,20 +418,27 @@ def adjust_brightness(img, brightness_factor):
...
+            :name: code-example1
+
+            >>> import numpy as np
+            >>> from PIL import Image
+            >>> from paddle.vision.transforms import functional as F
+            >>> np.random.seed(2023)
+
+            >>> fake_img = (np.random.rand(256, 300, 3) * 255.).astype('uint8')
+            >>> fake_img = Image.fromarray(fake_img)
+             >>> print(fake_img.size)
```
Comment (megemini) [inline python/paddle/vision/transforms/functional.py]: 多了一个空格 ～

File: python/paddle/vision/transforms/functional.py [hunk 9]
```diff
@@ -1039,35 +1059,32 @@ def erase(img, i, j, h, w, v, inplace=False):
...
+            >>> print(fake_img)
+            Tensor(shape=[3, 2, 4], dtype=float32, place=Place(cpu), stop_gradient=True,
+            [[[ 0.06132207,  1.11349595,  0.41906244, -0.24858207],
+            [-1.85169315, -1.50370061,  1.73954511,  0.13331604]],
+
+            [[ 1.66359663, -0.55764782, -0.59911072, -0.57773495],
+            [-1.03176904, -0.33741450, -0.29695082, -1.50258386]],
+
+            [[ 0.67233968, -1.07747352,  0.80170447, -0.06695852],
+            [-1.85003340, -0.23008066,  0.65083790,  0.75387722]]])
```
Comment (megemini) [inline python/paddle/vision/transforms/functional.py]: 输出不要空行 ～ 另外，可以对齐一下方括号 ～

``` python
Tensor(shape=[3, 2, 4], dtype=float32, place=Place(cpu), stop_gradient=True,
[[[ 0.06132207,  1.11349595,  0.41906244, -0.24858207],
  [-1.85169315, -1.50370061,  1.73954511,  0.13331604]],
 [[ 1.66359663, -0.55764782, -0.59911072, -0.57773495],
  [-1.03176904, -0.33741450, -0.29695082, -1.50258386]],
 [[ 0.67233968, -1.07747352,  0.80170447, -0.06695852],
  [-1.85003340, -0.23008066,  0.65083790,  0.75387722]]])
```

File: python/paddle/vision/transforms/functional.py [hunk 10]
```diff
@@ -1039,35 +1059,32 @@ def erase(img, i, j, h, w, v, inplace=False):
...
+            >>> print(result)
+            Tensor(shape=[3, 2, 4], dtype=float32, place=Place(cpu), stop_gradient=True,
+            [[[ 0.06132207,  0.        ,  0.        , -0.24858207],
+            [-1.85169315, -1.50370061,  1.73954511,  0.13331604]],
+
+            [[ 1.66359663,  0.        ,  0.        , -0.57773495],
+            [-1.03176904, -0.33741450, -0.29695082, -1.50258386]],
+
+            [[ 0.67233968,  0.        ,  0.        , -0.06695852],
+            [-1.85003340, -0.23008066,  0.65083790,  0.75387722]]])
```
Comment (megemini) [inline python/paddle/vision/transforms/functional.py]: 同上

File: python/paddle/vision/transforms/functional.py [hunk 11]
```diff
@@ -411,20 +418,27 @@ def adjust_brightness(img, brightness_factor):
...
-            from paddle.vision.transforms import functional as F
 
-            fake_img = (np.random.rand(256, 300, 3) * 255.).astype('uint8')
 
-            fake_img = Image.fromarray(fake_img)
-            print(fake_img.size) # (300, 256)
-            print(fake_img.load()[1,1]) # (95, 127, 202)
-            converted_img = F.adjust_brightness(fake_img, 0.5)
-            print(converted_img.size) # (300, 256)
-            print(converted_img.load()[1,1]) # (47, 63, 101)
```
Comment (megemini) [inline python/paddle/vision/transforms/functional.py:421]: 这里好多空行 ～ 是不是要整体修改一下 Codestyle ~

**Review by megemini — COMMENTED** [review]
有个地方写了很多空行，清理一下吧 ～ 其他的我这边没啥问题了 ～

**Review by SigureMo — APPROVED** [review]
LGTMeow 🐾
